infrastructure-playbooks/cephadm-adopt.yml

```
@@ -802,7 +788,6 @@
         name: ceph-rgw.target
```

```suggestion
        name: ceph-radosgw.target
```

I think that's a typo

```console
TASK [stop and disable ceph-radosgw systemd target] ******************************************************************************************************************************************************************************************
task path: /home/dim/ceph-ansible/infrastructure-playbooks/cephadm-adopt.yml:786
Tuesday 17 August 2021  00:53:17 +0200 (0:00:02.570)       0:10:45.285 ********
fatal: [rgw0]: FAILED! => changed=false 
  msg: 'Could not find the requested service ceph-rgw.target: host'
```